format-specs/geoparquet.md

```diff
@@ -52,13 +52,13 @@ Each geometry column in the dataset must be included in the columns field above
 
 | Field Name | Type | Description |
 | --- | --- | --- |
-| encoding | string | **REQUIRED** Name of the geometry encoding format. Currently only 'WKB' is supported. |
-| geometry_type | string or \[string] | **REQUIRED** The geometry type(s) of all geometries, or 'Unknown' if they are not known. |
-| crs | JSON object | **OPTIONAL** [PROJJSON](https://proj.org/specifications/projjson.html) JSON object representing the Coordinate Reference System (CRS) of the geometry. If the crs field is not included then the data in this column must be stored in longitude, latitude based on the WGS84 datum, and CRS-aware implementations should assume a default value of [OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84). |
-| orientation | string | **OPTIONAL** Winding order of exterior ring of polygons. If present must be 'counterclockwise'; interior rings are wound in opposite order. If absent, no assertions are made regarding the winding order. |
-| edges | string | **OPTIONAL** Name of the coordinate system for the edges. Must be one of 'planar' or 'spherical'. The default value is 'planar'. |
+| encoding | string | **REQUIRED** Name of the geometry encoding format. Currently only `WKB` is supported. |
+| geometry_type | string or \[string] | **REQUIRED** The geometry type(s) of all geometries, or `Unknown` if they are not known. |
+| crs | object | **OPTIONAL** [PROJJSON](https://proj.org/specifications/projjson.html) JSON object representing the Coordinate Reference System (CRS) of the geometry. If the crs field is not included then the data in this column must be stored in longitude, latitude based on the WGS84 datum, and CRS-aware implementations should assume a default value of [OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84). |
+| orientation | string | **OPTIONAL** Winding order of exterior ring of polygons. If present must be `counterclockwise`; interior rings are wound in opposite order. If absent, no assertions are made regarding the winding order. |
+| edges | string | **OPTIONAL** Name of the coordinate system for the edges. Must be one of `planar` or `spherical`. The default value is 'planar'. |
 | bbox | \[number] | **OPTIONAL** Bounding Box of the geometries in the file, formatted according to [RFC 7946, section 5](https://tools.ietf.org/html/rfc7946#section-5). |
-| epoch | double | **OPTIONAL** Coordinate epoch in case of a dynamic CRS, expressed as a decimal year. |
+| epoch | number | **OPTIONAL** Coordinate epoch in case of a dynamic CRS, expressed as a decimal year. |
```

Comment:
This is allowed to be an integer, right?

Reply:
It should be a float, in theory, but I suppose implementations won't care if it's rounded?

Reply:
We could in theory enforce a float, but I've never seen this before in JSON Schema...

Reply:
Would be something like this:
```json
{
  "type": "number",
  "not": {
  	"type": "integer"
  }
}
```

Reply:
I would avoid gymnastics like this.  Instead stick with JSON types when using JSON.

Reply:
number, allowing integer or floating-point, should be just fine

Reply:
Yeah, doesn't even work because 7.0 is a valid integer according to JSON schema. :-)